pkg/server/cstorvolumeconfig/https_test.go

```diff
@@ -182,11 +186,11 @@ func (f *fixture) createFakeVolumeReplicas(
 			"cstorvolume.openebs.io/name":              volumeName,
 		}
 		cvr := cstor.NewCStorVolumeReplica().
-			WithName(volumeName + "-" + cspiList.Items[0].Name).
+			WithName(volumeName + "-" + cspiList.Items[i].Name).
 			WithLabelsNew(labels).
 			WithStatusPhase(phase)
 		_, err := f.openebsClient.CstorV1().CStorVolumeReplicas(namespace).Create(cvr)
-		if err != nil {
+		if err != nil && !k8serror.IsNotFound(err) {
```

Comment:
why there will be not found error while create ?

Reply:
It shouldn't be`IsAlreadyExist` somehow copy-pasted the wrong check